### PR TITLE
Update bash completions

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

File: tools/check_man.py [hunk 1]
```diff
@@ -182,11 +184,15 @@ def parse_cmdline():
                         help="man-page in mdoc format")
     parser.add_argument("-c", "--check-file",
                         help="check that all options are in the file")
+    parser.add_argument("--edit-bash",
```
Comment (cperciva) [inline tools/check_man.py:187]: Should "edit" be "regenerate"?

Comment (gperciva) [reply tools/check_man.py:187]: Hmm.  It certainly doesn't generate it from scratch, but I suppose that "regenerate" could work.

Comment (gperciva) [reply tools/check_man.py:187]: ... or maybe "update", since that's what I ended up calling the branch?

**Comment by gperciva** [comment]
Updated

**Pull request merged**